infra/templates/az-isolated-service-single-region/terraform.tfvars

```diff
@@ -30,7 +30,19 @@ unauthn_deployment_targets = [
 
 # Note: this is configured as such only to test IP Whitelists. This is a well
```

Comment:
The comment was out-of-date before, but can we update it to reflect what these IPs are?

My *hunch* is that they are the build agent IPs that we are using in our build. However, its difficult to determine from the context alone.